src/sessionInfo/SessionCreator.ts

```diff
@@ -50,53 +48,55 @@ export default class SessionCreator implements ISessionCreator {
     @inject("storageUtility") private storageUtility: IStorageUtility
   ) {}
 
-  create(options: ISessionCreatorOptions): ISolidSession {
-    const localUserId: string = options.localUserId || this.uuidGenerator.v4();
-    if (options.loggedIn) {
-      return {
-        localUserId,
-        loggedIn: true,
-        webId: options.webId as string,
-        neededAction: options.neededAction || { actionType: "inaction" },
-        state: options.state,
-        logout: async (): Promise<void> => {
-          // TODO: handle if webid isn't here
-          return this.logoutHandler.handle(localUserId);
-        },
-        fetch: (url: RequestInfo, init?: RequestInit): Promise<Response> => {
-          // TODO: handle if webid isn't here
-          return this.authenticatedFetcher.handle(
-            {
-              localUserId,
-              type: "dpop"
-            },
-            url,
-            init
-          );
-        }
-      };
-    } else {
-      return {
-        localUserId,
-        loggedIn: false,
-        neededAction: options.neededAction || { actionType: "inaction" }
-      };
-    }
+  create(options: ISessionCreatorOptions): ISessionInfo {
+    // const localUserId: string = options.localUserId || this.uuidGenerator.v4();
```

Comment:
Ditto

Reply:
Partly resolved in 378c0fb59b9a2808fb72b67fcc05e72532688441, but still some commented-out code that needs to be fixed there. Should be adressed in a different PR.